### PR TITLE
[bitnami/kong] Chart standardized

### PR DESCRIPTION
**Pull request by juan131**
Signed-off-by: juan131 <juanariza@vmware.com>

**Description of the change**

This PR standardizes the `bitnami/kong` chart to be in line with the rest of the catalog.

**Benefits**

Standardization.

**Possible drawbacks**

None. No major bump is required since breaking changes were added support for deprecated values using `coalesce`.

**Applicable issues**

N/A

**Additional information**

N/A

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the values.yaml and added to the README.md using (readme-generator-for-helm)[https://github.com/bitnami-labs/readme-generator-for-helm]
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/kong/templates/metrics-service.yaml [hunk 1]
```diff
@@ -18,19 +19,11 @@ metadata:
     {{- end }}
   {{- end }}
```
Comment (javsalgar) [inline bitnami/kong/templates/metrics-service.yaml:20]: Missing `metrics.service.type`

Comment (javsalgar) [reply bitnami/kong/templates/metrics-service.yaml:20]: Also missing references to other standard service elements.

**Review by javsalgar — CHANGES_REQUESTED** [review]
Thank you! Changes requested

File: bitnami/kong/values.yaml [hunk 2]
```diff
@@ -219,13 +242,124 @@ autoscaling:
         targetAverageUtilization: 80
 ## Kong Pod Disruption Budget
 ## ref: https://kubernetes.io/docs/concepts/workloads/pods/disruptions/
-## @param pdb.enabled Deploy a pdb object for the Kong pod
+## @param pdb.enabled Deploy a PodDisruptionBudget object for Kong deployment
+## @param pdb.minAvailable Minimum available Kong replicas (expressed in percentage)
 ## @param pdb.maxUnavailable Maximum unavailable Kong replicas (expressed in percentage)
 ##
 pdb:
   enabled: false
```
Comment (migruiz4) [inline bitnami/kong/values.yaml:250]: To be aligned with other charts, could you please rename this as `pdb.create`?

**Review by migruiz4 — CHANGES_REQUESTED** [review]
On my side, just a small suggestion, but please check existing suggestions.

File: bitnami/kong/templates/metrics-service.yaml (hunk 1 above)
Comment (juan131) [reply bitnami/kong/templates/metrics-service.yaml:20]: Hardcoded to use a ClusterIP service.

Comment (juan131) [reply bitnami/kong/templates/metrics-service.yaml:20]: The rest of parameters that make sense were added.

**Review by migruiz4 — DISMISSED** [review]

**Review by javsalgar — DISMISSED** [review]

**Comment by davidchriqui** [comment]
@juan131   May I ask why having constrained the use of an arbitrary service.clusterIP only when service.type is "ClusterIP"?

**Pull request merged**